src/main.ts

```diff
@@ -169,7 +173,7 @@ function boxClickHandler(e: MouseEvent): void {
 }
 
 function getBoxHeaders(): NodeListOf<HTMLDivElement> {
-  return $$('#imagelist-container .media-box__header, #imagelist_container .media-box__header');
+  return $$('#imagelist-container > .media-list .media-box__header, #imagelist_container > .media-list .media-box__header');
```

Comment:
The addition of `> .media-list` is redundant here, the original selector already excludes featured and trending images.

Reply:
Intent was to make selector similar to one in `toggleUI()`